src/MainNav/MainNav.test.js

```diff
@@ -15,7 +15,16 @@ describe('MainNav', () => {
         />
       </Router>
     )
+  })
+  it('should render a title and image', () => {
     expect(screen.getByAltText('National Parkfinder Logo')).toBeInTheDocument()
     expect(screen.getByText('National Parkfinder')).toBeInTheDocument()
   })
-})
\ No newline at end of file
+
+  it('should render a list of park buttons', () => {
+    expect(screen.getByText('Grand Canyon')).toBeInTheDocument()
+    const allButtons = screen.getAllByRole("button")
+    expect(allButtons.length).toBe(56)
```

Comment:
This was a cool solution to ensuring that all of the buttons from our local data are being rendered

Reply:
Yes! I agree with Bailey!